users/user_files.sls

```diff
@@ -28,7 +28,7 @@ include:
 {%- if not skip_user %}
 users_userfiles_{{ username }}_recursive:
   file.recurse:
-    - name: {{ user.home }}
+    - name: {{ salt['pillar.get']( username ~ 'user:home', '/home/' ~ username )}}
```

Comment:
Sure I used exactly same code in my "refactored" branch https://github.com/hatifnatt/users-formula/blob/hatifnatt-user_files.sls/users/user_files.sls#L10